LightBlue.MultiHost/Runners/RoleRunner.cs

```diff
@@ -46,6 +47,15 @@ public RoleRunner(Role role)
 
         public void Start()
         {
+            if (App.Configuration.CustomRunners.Any(x => x.Name == _role.RoleName))
+            {
+                var customRunner = App.Configuration.CustomRunners.First(x => x.Name == _role.RoleName);
+                var role = RunnerFactory.CreateCustomRunner(customRunner, _role);
+                _resources.Add(role);
+                role.Start();
+                return;
+            }
```

Comment:
nit: should be able to do this with one scan of the enumerable

```suggestion
            if (App.Configuration.CustomRunners.FirstOrDefault(x => x.Name.Equals(_role.RoleName,  StringComparison.OrdinalIgnoreCase)) is CustomRunner customRunner)
            {
                var role = RunnerFactory.CreateCustomRunner(customRunner, _role);
                _resources.Add(role);
                role.Start();
                return;
            }
```